<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HamWindow</class>
 <widget class="QWidget" name="HamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="UpButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>721</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DownButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ScanButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
